src/routes/events.js

```diff
@@ -62,6 +86,18 @@ module.exports = (app) => {
 
   app.get('/upcoming-high-priority-and-nearby', async ({ success, failed, event, context }) => {
     context.callbackWaitsForEmptyEventLoop = false;
+
+    const {
+      requestContext: {
+        identity: {
+          sourceIp,
+          userAgent,
+        },
+      },
+    } = event;
+
+    console.log('Incoming request', { path: '/upcoming', sourceIp, userAgent });
```

Comment:
`/upcoming-high-priority-and-nearby` 
Might be useful to log params too. I wonder if people are scraping with different lat/lngs. Maybe they don't realize that MobilizeAmerica has a public API that will return all events nationwide with pagination...

Reply:
Oo yeah I can add that